.ci_support/osx_64_.yaml

```diff
@@ -18,8 +18,6 @@ lz4_c:
 - 1.9.2
 macos_machine:
 - x86_64-apple-darwin13.4.0
-macos_min_version:
-- '10.9'
```

Comment:
Why are we removing this?

Reply:
We didn't. @isuruf merged the change in the global pinnings file and smithy did. He can comment on that change.

Reply:
Smithy did that, not me :-)

Reply:
I think I know the origin of that change and I am completely fine with it.

Reply:
What do you mean?

Reply:
@beckermr I was referring to the discussion at https://github.com/conda-forge/pyside2-feedstock/issues/46#issuecomment-563911075